src/OrbitVulkanLayer/VulkanLayerControllerTest.cpp

```diff
@@ -325,18 +332,45 @@ TEST_F(VulkanLayerControllerTest, InitializationFailsOnCreateInstanceWithNoInfo)
   EXPECT_EQ(result, VK_ERROR_INITIALIZATION_FAILED);
 }
 
-TEST_F(VulkanLayerControllerTest,
-       WillCreateDispatchTableAndVulkanLayerProducerAndAdvanceLinkageOnCreateInstance) {
+TEST_F(
+    VulkanLayerControllerTest,
+    WillCreateDispatchTableAndVulkanLayerProducerAndAdvanceLinkageOnCreateInstanceWithGameNotEnablingRequiredExtensions) {
```

Comment:
Maybe the name of the test is long enough by now that a name should be added.

Reply:
What do you mean by "a name should be added"? I don't quite get it. (But for sure, the name is long).

Reply:
Sorry, I meant a comment.

Reply:
Ah okay, that makes sense. I will address your comments (including this) tomorrow morning. Thanks for them!

Reply:
Done.